articles/machine-learning/tutorial-1st-experiment-sdk-setup-local.md

```diff
@@ -136,7 +136,8 @@ try:
     print('Found existing cluster, use it.')
 except ComputeTargetException:
     compute_config = AmlCompute.provisioning_configuration(vm_size='STANDARD_D2_V2',
```

Comment:
Since we're being explicit, should we also add `min_nodes=0` as an argument?

Reply:
We can if we want to, and max_node is required to run successfully.